f5_openstack_agent/lbaasv2/drivers/bigip/agent.py

```diff
@@ -33,13 +33,19 @@
 
 
 try:
-    from neutron.agent.linux import interface
     from neutron.common import config as common_config
     from neutron.common import rpc as n_rpc
     try:
         from neutron.conf.agent import common as config
     except Exception:
         from neutron.agent.common import config
+    # pzhang(NOTE): for openstack backward compatible
+    try:
+        # m/o/p version neutron config opts
+        from neutron.agent.linux.interface import OPTS as INTERFACE_OPTS
+    except Exception:
+        # q version neutron config opts
+        from neutron.conf.agent.common import INTERFACE_OPTS
```

Comment:
Should we put recent version in `try`, and make it compatible to m/o/p in `except`?